my-pern-app/backend/src/server.ts

```diff
@@ -44,10 +44,10 @@ app.put('/api/updateMovie/:movieId', async (req, res, next)=>{
         const {newTitle, newSummary, newImdbLink, newRating} = req.body
         const sql = `
         update "movies"
-        set "title" = $2
```

Comment:
For some reason, the PR isn't letting me comment on every line. This comment is for lines 3 and 7.

The preferred way to use `dotenv` is:
`import 'dotenv/config';`

Then you do _not_ call `dotenv.config();`.
Further, the above import _must_ be the first line in your main file so that all imported files have access to the environment variables.